packages/web3-validator/src/errors.ts

```diff
@@ -0,0 +1,30 @@
+import { Web3ValidationErrorObject } from './types';
```

Comment:
should this be apart of the `errors.ts` in web3-common, so we have are pulling errors from one source?

Reply:
For one type should not add a dependency of the package. In future we should create `web3-errors` then extract all errors there.